packages/addons/src/plugins/autoHeightTextarea.ts

```diff
@@ -12,16 +12,26 @@ export function createAutoHeightTextareaPlugin(): FormKitPlugin {
   const autoHeightTextareaPlugin = (node: FormKitNode) => {
     if (node.props.type !== 'textarea') return
     node.addProps(['autoHeight'])
+    node.addProps(['maxAutoHeight'])
```

Comment:
These two lines can be combined.

```suggestion
    node.addProps(['autoHeight', 'maxAutoHeight'])
```